storagemanager/storage/migrations/0003_auto_20200928_1620.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.1 on 2020-09-28 14:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('storage', '0002_auto_20200928_1535'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='item',
+            name='category',
+            field=models.CharField(choices=[('Sound', 'Sound'), ('Light', 'Light'), ('Stage', 'Stage'), ('Power', 'Power'), ('Extra', 'Extra'), ('bonus', 'Bonus')], default='Sound', max_length=10),
```

Comment:
It looks like you added some changes, run `makemigrations` for 0002 and then changed your mind and did that again to generate 0003, right? If so, as 0002 wasn't done in a separate PR, it might be better to just delete both migration files and run `makemigrations` again, so you will end up with only 0002 migration with the latest changes.

Also, it is a good practice to rename migration files to some descriptive naming (but leave their numbers as it is), so i.e. `0002_item_category_field_as_choices.py`